### ISSUE TITLE
USB-C video output does not work

### ISSUE THREAD

**Issue by ZambraDesigner** [daliansky/XiaoMi-Pro#176]
I connected my LG Monitor (https://www.lg.com/it/monitor/lg-29UM59A) via USB-C but it doesn't work ... on the warning it tells me that there is no Video signal.
How can I do? I used the original power cord.

If I connect it via HDMI only the video works and the audio does not pass.

**Comment by LisztFU** [comment]
Did you try to use a cable over thunderbolt 3? With the USB2.0, video out can't work.

**Comment by ZambraDesigner** [comment]
I used the original cable with which I charge it.
Neither video nor energy recharge worked

**Comment by LisztFU** [comment]
'Cause the stock cable only supports USB2.0. Go get some quality Type-C which is 3.0 compatible. Notice that the original MacBook cable is USB2.0 only.

**Comment by ZambraDesigner** [comment]
OK thanks.
the 4k output is already enabled on the type-c port or do you have to enable it as for HDMI following this guide?
https://github.com/daliansky/XiaoMi-Pro/wiki/Set-DVMT-to-64mb

I have the BIOS A07 and it gives me an error that does not find the file on the first step to backup

**Comment by shilch** [comment]
Did you try a lower resolution? Might be related to #117

**Comment by ZambraDesigner** [comment]
> Did you try a lower resolution? Might be related to #117

I've tried, the conflict between HDMI and USB 3 I have with all the resolutions.
A first solution was to connect the Wifi USB to a USB 2.0 HUB and in this way HDMI and USB port with HUB do not conflict.

Today I got this cable https://amzn.to/2HsiYMD, and finally I can use the monitor in 4K @ 60Hz;) and it doesn't create any conflict with the USB Wifi.
PS: this resolution is activated with active HDPI and after waking Xiaomi from sleep. ([as explained here](https://github.com/daliansky/XiaoMi-Pro/issues/117#issuecomment-493079721))

Excellent solution.

**Comment by Raphencoder** [comment]
So does the USB type C 3.0 works for video signal ?

**Comment by ZambraDesigner** [comment]
> So does the USB type C 3.0 works for video signal ?

Yes, the type-c you use for the top-up also works as a video output. The other hand is only for data exchange but not video.
